batch/batch/front_end/front_end.py

```diff
@@ -2213,18 +2214,18 @@ async def _commit_update(app: web.Application, batch_id: int, update_id: int, us
 
 
 @routes.delete('/api/v1alpha/batches/{batch_id}')
+@api_security_headers
 @billing_project_users_only()
 @add_metadata_to_request
-@api_security_headers
 async def delete_batch(request: web.Request, _, batch_id: int) -> web.Response:
     await _delete_batch(request.app, batch_id)
     return web.Response()
 
 
 @routes.get('/batches/{batch_id}')
+@web_security_headers_unsafe_eval
```

Comment:
These are mainly required because of the `x-...` annotations in the html (eg `<li x-data='{open: {{ "true" if starts_open else "false" }}}'>` in `metadata_tables.html`, imported by both `job` and `batch` details pages